pipeline/translate/bestbleu.py

```diff
@@ -96,7 +96,7 @@ def marian_best_bleu(args, score_function):
 
 def compute_chrf(references, translation):
     hypo = " ".join(translation)
-    refs = [" ".join(r) for r in references][0]
+    refs = [" ".join(r) for r in references]
```

Comment:
Suggestion (optional): It would be nice to have a `run_task` test for this file if we're changing code. I'm not sure how we know this code is correct otherwise. I'm marking as optional since it's not changing much here.